src/eko/evolution_operator/grid.py

```diff
@@ -276,6 +278,19 @@ def generate(self, q2):
                 )
                 final_op = final_op @ rot @ matching @ phys_op
 
+        # Compute scale variation scheme B
+        if self.config["SV_scheme"] == "B":
+            scale_var = ScaleVariationOperator(
+                self.config, self.managers, path[-1].nf, final_op.q2_final
+            )
+            scale_var.compute()
+            scale_var_op = physical.PhysicalOperator.ad_to_evol_map(
+                scale_var.op_members,
+                scale_var.nf,
+                scale_var.q2_to,
+                intrinsic_range,
+             )
+            final_op = scale_var_op @ final_op
```

Comment:
I wonder whether we need to do this path by path - instead of only at the end ... see scheme A: here we're affected patch by patch and I think we have to continue to do so; should we ask for a confirmation?

Reply:
Uhm... I'd say that what we split from the exponential is only in the last patch, because the patches are in the argument of the exponential itself.
But we might discuss.

Reply:
mmm but the actual SV kernel depends on `nf`, so it can make differences